test/fixtures/with_config/.netlify/edge-functions/framework-func1.ts

```diff
@@ -7,5 +7,5 @@ export default async () => {
 }
 
 export const config = () => ({
-  path: '/framework-func1',
```

Comment:
Is there a reason to change all of these to the array format? Also, there's something weird going on with the formatting (see the missing space after `path:`).

Reply:
I'll change them back to a string, as I think a string will probably still be used more often, and we have tests in place to make sure an array would work as well.